packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2352,16 +2352,30 @@ class RenderFittedBox extends RenderProxyBox {
     markNeedsPaint();
   }
 
+  bool _fitAffectsLayout(BoxFit fit) {
```

Comment:
Small change but;
```diff
+ return fit == BoxFit.scaleDown;
-   switch (fit) {
-      case BoxFit.scaleDown:
-        return true;
-      default:
-        return false;
-    }
-  }
```

Reply:
switch cases are generally preferred over comparison in the repo, but so is the default case tbh

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or--or--with-enum-values